### PR TITLE
Migration script for Quarkus 3 Alpha 1

### PR DESCRIPTION
**Pull request by ia3andy**
- This script has been tested with success on the Quarkus quickstarts
- It is experimental and is meant to be integrated in a smooth Quarkus tooling experience 
- It also work to migrate extensions
- Part of https://github.com/quarkusio/quarkus/issues/29000

## Usage

Download the quarkus3.yml migration script on your local machine and execute this command in the project or extension you want to migrate:
```shell
mvn org.openrewrite.maven:rewrite-maven-plugin:4.36.0:run \
   -Drewrite.configLocation=[SOME_PATH]/quarkus3.yml \
   -DactiveRecipes=io.quarkus.openrewrite.Quarkus3
```

### CODE REVIEW

File: jakarta/quarkus3.yml [hunk 1]
```diff
@@ -0,0 +1,984 @@
...
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.Quarkus3
```
Comment (gsmet) [inline jakarta/quarkus3.yml:18]: Minor but this should probably be under the `io.quarkus` namespace given it's a Quarkus-specific recipe.

Maybe `io.quarkus.openrewrite.migrate.Quarkus3`?

File: jakarta/quarkus3.yml [hunk 2]
```diff
@@ -0,0 +1,984 @@
...
+  - org.openrewrite.java.migrate.JavaxMigrationToJakarta
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: quarkus.platform.version
+      newValue: 3.0.0.Alpha1
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: quarkus.version
+      newValue: 3.0.0.Alpha1
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.JavaxMigrationToJakarta
```
Comment (gsmet) [inline jakarta/quarkus3.yml:33]: You have to copy the recipe here? You cannot refer to the one bundled in OpenRewrite (I suppose it's bundled)? Just asking, I have no idea if it works.

**Review by gsmet — COMMENTED** [review]
Cool, thanks! Added a small suggestion and a question.

File: jakarta/quarkus3.yml (hunk 2 above)
Comment (ia3andy) [reply jakarta/quarkus3.yml:33]: can you give it a try, it works locally, but maybe I have it somewhere already..

Comment (ia3andy) [reply jakarta/quarkus3.yml:33]: @gsmet sorry I misunderstood you, no you can't refer to the other one AFAIK. Also, I've add to comment a few stuff inside it as it doesn't apply to Quarkus (Add dependencies)

**Comment by ia3andy** [comment]
@gsmet fixed

**Review by gsmet — APPROVED** [review]
I was able to convert my Quarkus GitHub App extension to 3.0.0.Alpha1 and Jakarta packages with this command so it looks good enough to me.

**Comment by holly-cummins** [comment]
When merged, will this script have a stable URL, or should we just refer to this branch?

**Pull request merged**